vitest.config.js

```diff
@@ -0,0 +1,32 @@
+import { defineConfig } from 'vite'
+import Vue from '@vitejs/plugin-vue'
+const path = require('path')
+
+const configTest = defineConfig({
+  plugins: [Vue()],
+  resolve: {
+    alias: [
+      {
+        find: '@',
+        replacement: path.resolve(__dirname, '/src'),
```

Comment:
This alias doesn't work on my local 
I had replacing it with:  

import { fileURLToPath, URL } from 'node:url'
replacement:` fileURLToPath(new URL('./src', import.meta.url))

Does this change also work on your local?
Thanks